cmd/image-converter/image-converter.go

```diff
@@ -57,21 +72,5 @@ func convert(srcVolPath, dstVolPath, sourceFormat, targetFormat string) error {
 		return err
 	}
 
-	// Replace /mnt/disk.img with /output/disk.img
-	cmd = exec.Command(
```

Comment:
why mv or cp can't work while qemu-img convert does?

Reply:
it works only for files, for block volumes you need to copy the content of the device

although now that I think about it may actually make sense to keep the `mv` for files, because it should be much faster

Reply:
> although now that I think about it may actually make sense to keep the `mv` for files, because it should be much faster

are you sure qemu-img doesn't have this optimization? AFAIU vdsm always invokes qemu-img for this and I find it surprising if it could have been optimized

Reply:
I suspect that's because of the other parameters vdsm uses, I'd be very surprised if qemu-img will do what `mv` does